internal/core/command/types.go

```diff
@@ -15,6 +15,7 @@ package command
 
 import (
 	"bytes"
+	"fmt"
```

Comment:
Please sort imports

Reply:
Imports sorted, the test TestExecuteWithLockedDevice - removed.
I have removed this test,  because device.AdminStatus validation is done earlier, immediately after the device is Loaded. So, the test does not return anymore Error because of the device state.